tests/alembic/revisions/test_e9b705f9adf9_add_server_defaults_to_timestamp_.py

```
@@ -0,0 +1,284 @@
+    ]:
+        columns = inspector.get_columns(table, schema=schema)
+        col_info = next((c for c in columns if c["name"] == column), None)
+        assert (
+            col_info is not None
+        ), f"Column {table}.{column} should exist before migration"
+
+    # Create test data to ensure existing rows work after migration
+    conn.execute(
+        text(
+            f'INSERT INTO "{schema}"."workspaces" '
+            + '("id", "name", "created_at", "metadata", "internal_metadata", "configuration") '
```

Ruff (0.14.2)

Explicitly concatenated string should be implicitly concatenated

Remove redundant '+' operator to implicitly concatenate

(ISC003)